sql/table.sql

```diff
@@ -78,8 +79,8 @@ CREATE TABLE genomic_feature_disease (
   genomic_feature_id int(10) unsigned NOT NULL,
   disease_id int(10) unsigned NOT NULL,
   original_allelic_requirement_attrib set('1','2','3','4','5','6','7','8','9','10','11','12','13','14','15','16','17','18','19','20') DEFAULT NULL,
-  allelic_requirement_attrib set('59','60','61','62','63','64','65','66','67','68','69','70') DEFAULT NULL,
```

Comment:
The patch is only for cross_cutting_modifier_attrib. 
Any change to allelic_requirement_attrib should be done in a separate PR.